config/webpack.config.dev.js

```diff
@@ -65,6 +65,9 @@ module.exports = {
     publicPath: '/'
   },
   resolve: {
+    // This allows you to set a root for where webpack should look for modules.
+    // This enables you to use absolute imports from the root.
+    root: path.resolve(env['process.env.NODE_PATH']),
```

Comment:
I wasn't sure the exact reason for `env.js`, but that makes sense and I've removed it from there and just accessed it directly.

It does not currently support multiple paths. Would that be something you want me to add in this PR?